scripts/app.js

```diff
@@ -1,3 +1,8 @@
+// Redirect if the URL contains an empty search query
+const urlParams = new URLSearchParams(window.location.search);
```

Comment:
variable `urlParams` is also declared in [another line](https://github.com/oyepriyansh/DevProfiles/blob/9d598168523bfb3e88fd75bd5460f5c620cab0df/scripts/app.js#L147) as const